src/components/molecules/Slider/index.js

```diff
@@ -1,67 +1,79 @@
 import React, { useState } from 'react';
 import Button from '../../atoms/Button';
-import img1 from '../../../images/carousel_img_1.png';
-import img2 from '../../../images/carousel_img_2.png';
-import img3 from '../../../images/carousel_img_3.png';
+import CarouselText from '../CarouselText';
 import './style.scss';
 
-const Slider = () => {
-  const img = [
-    <img key={img1} src={img1} alt="img1" />,
-    <img key={img2} src={img2} alt="img2" />,
-    <img key={img3} src={img3} alt="img3" />,
+const Slider = ({ title, subtitle, images }) => {
+  const slides = [
+    <img key="1" src={images && require(`../../../images/${images[2].src}`).default} alt={images && images[2].alt} />,
+    <img key="2" src={images && require(`../../../images/${images[0].src}`).default} alt={images && images[0].alt} />,
+    <img key="3" src={images && require(`../../../images/${images[1].src}`).default} alt={images && images[1].alt} />,
+    <img key="4" src={images && require(`../../../images/${images[2].src}`).default} alt={images && images[2].alt} />,
+    <img key="5" src={images && require(`../../../images/${images[0].src}`).default} alt={images && images[0].alt} />,
```

Comment:
Why do you have duplicate images here?

Reply:
This is how this carousel works: I created a duplicate of the first and a duplicate of the last slide, and when I move to them, I jump to the first or last slide without animation. This is for a smooth transition between the start and end of the carousel.